staticconf/config.py

```diff
@@ -415,23 +415,44 @@ def has_changed(self):
         return last_inodes != self.inodes
 
 
+def build_compare_func(err_logger=None):
+    """Returns a compare_func that can be passed to MTimeComparator.
+
+    The returned compare_func first tries os.path.getmtime(filename),
+    then calls err_logger(filename) if that fails. If err_logger is None,
+    then it does nothing. err_logger is always called within the context of
+    an OSError raised by os.path.getmtime(filename). Information on this
+    error can be retrieved by calling sys.exc_info inside of err_logger."""
+    def compare_func(filename):
+        try:
+            return os.path.getmtime(filename)
+        except OSError:
+            if err_logger is not None:
+                err_logger(filename)
+        return -1
+    return compare_func
```

Comment:
This function has very little to do with the rest of the module, but I guess it's ok to keep it.